src/components/groups/OverView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { PiStudentFill } from "react-icons/pi";
+import { MdGroups2 } from "react-icons/md";
+import { SiZenn } from "react-icons/si";
+import OverViewCard, { CardProps } from "../common/OverViewCard";
+import {
+  useGetGroupTotalStudentQuery,
+  useGetTotalGroupsQuery,
+  useGetGroupMeanSizeQuery,
+} from "<@>/store/groups/groups-api";
+import Loading from "../common/Loading";
+import FetchingError from "../common/FetchingError";
+
+const GroupOverView: React.FC = () => {
+  const { data: students, isLoading, error } = useGetGroupTotalStudentQuery();
+  const { data: groups, isLoading: isLoadingGroups } = useGetTotalGroupsQuery();
+  const { data: meanSize, isLoading: isLoadingMeanSize } =
+    useGetGroupMeanSizeQuery();
+    
+  const GroupOverViewData: CardProps[] = [
```

Comment:
The type `CardProps[]` shouldn't contain a suffix called props since it is not a props.